targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/lp_ticker.h

```diff
@@ -0,0 +1,27 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef LP_TICKER_H
+#define LP_TICKER_H
+
+/* TIMER0 - SoftDevice.
+ * TIMER1 - us ticker.
+ * TIMER2 - lp ticker. */
+
+#define LP_TICKER_COUNTER_BITS        32u
+#define LP_TICKER_FREQ                31250
+
+#endif // LP_TICKER_H
```

Comment:
I can't see where this file is being used?

Reply:
This file is added by mistake. I have removed it.